keras_retinanet/models/retinanet.py

```diff
@@ -273,10 +301,54 @@ def retinanet(
     if submodels is None:
         submodels = default_submodels(num_classes, num_anchors)
 
-    C3, C4, C5 = backbone_layers
+    if pyramid_levels is None:
+        pyramid_levels = [3,4,5,6,7]
+
+
+    C2, C3, C4, C5 = None, None, None, None
+
+    backbone_layer_counter = 0
+    backbone_layer_number = len(backbone_layers)
+
 
+    if backbone_layer_number < 3:
+        raise ValueError('Must provide at least 3 layers for backbone')
+
+    if backbone_layer_number < 4 and 2 in pyramid_levels:
+        raise ValueError("backbone model providing {} layers. Need 4 backbone layers to use pyramid level 2 layers.\
```

Comment:
This kind of string concatenation doesn't work:
```python
In [1]: print("Hello \ 
   ...:     world")                                                                                                                                                                                                                            
Hello     world
```

Reply:
oh oops yeah. It's all one line now and since the non-resnet warnings are gone it's not too too long anymore